README.md

```diff
@@ -681,3 +437,120 @@ thereafter it should be possible to use one of the different checkpatch targets
 in the [Makefile](Makefile). There are targets for checking all files, checking
 against latest commit, against a certain base-commit etc. For the details, read
 the [Makefile](Makefile).
+
+## 6. repo manifests
```

Comment:
Should not we link to https://github.com/OP-TEE/manifest instead of having a kind of copy/paste?